test/Sentry.DiagnosticSource.IntegrationTests/SqlListenerTests.cs

```diff
@@ -14,11 +14,11 @@ static SqlListenerTests()
             buildTemplate: TestDbBuilder.CreateTable);
     }
 
+#if !NET48
```

Comment:
- nit: `#if !NETFRAMEWORK` would be better, so the framework version is only in the csproj
- q: Why is it that we can't run this test on .NET Framework anyway? I see that if I try, it doesn't capture any spans.  Does the SqlClient not implement `DiagnosticListener` on .NET Framework implementation?  I hunted, but couldn't confirm that.  I thought the diagnostic source stuff was available across the board, perhaps not?

Reply:
> #if !NETFRAMEWORK

yep. it is better

> Why is it that we can't run this test on .NET Framework anyway

https://github.com/dotnet/SqlClient/issues/1529